app/controllers/stories_controller.rb

```diff
@@ -61,7 +61,7 @@ def update
   def destroy
     @story = policy_scope(Story).find(params[:id])
     authorize @story
-    StoryOperations::Destroy.call(@story, current_user)
+    StoryOperations::Destroy.new.call(story: @story, current_user: current_user)
```

Comment:
Hey there, one thing that I'm not geting is why here you do `call(story: @story, current_user: current_user)` but in the `StoryOperations::Destroy` class the method call has a signature of just `def call(params)`. This is right?

Reply:
passing params like `call(story: @story, current_user: current_user)` turns them into a hash, thats why I can use `params[]` inside the call.

Reply:
Sure, but I think this a bit difficult to read while debugging. Heres my suggestion, you change the method signature to use named_params (same as you're using here), this way you won't need to do `params.fetch(:story` for instance, you can use the argument directly in your method. What do you think about this? Let me know. Nice work btw.

Reply:
agreed and done! :rocket: